.github/workflows/manual-deploy.yml

```
@@ -0,0 +1,361 @@
+      - name: Install Dependencies
+        working-directory: backend
+        run: npm install
+
+      - name: Build Application
+        working-directory: backend
+        run: npm run build
+        env:
+          NODE_ENV: ${{ github.event.inputs.environment }}
+
+      - name: Generate Version ID
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)